src/OpenTelemetry.Extensions/OpenTelemetry.Extensions.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoRoot)\build\CodeAnalysis.props" />
+
+  <PropertyGroup>
+    <TargetFrameworks>net452;net461;netstandard2.0</TargetFrameworks>
```

Comment:
we may avoid net452 now, as currently this is only having a Ilogger related functionality.
If there is a need, we can add net452 in the future.

Reply:
Done (on contrib PR)